sdk/src/Services/SQS/Custom/Model/GetQueueAttributesResponse.extensions.cs

```diff
@@ -178,6 +172,44 @@ public string Policy
             }
         }
 
+        /// <summary>
+        /// Whether or not the queue is a First-In-First-Out (FIFO) queue
+        /// </summary>
+        public bool FifoQueue
+        {
+            get
+            {
+                var attributeValue = getAttributeValue(SQSConstants.ATTRIBUTE_FIFO_QUEUE);
+
+                if (attributeValue == null || attributeValue.Trim() == string.Empty)
+                {
+                    return false;
+                }
+
+                bool result;
+                return bool.TryParse(attributeValue, out result);
```

Comment:
This works with the current service behavior: if the service returns *anything* for this attribute, it will be `true`, otherwise the service throws an exception.
If the service changes behavior in the future and does not throw an exception for non-FIFO queues and does return `true` and `false` values, this code will always return "true".

The safer approach here is to do this:
```
bool result;
if (bool.TryParse(attributeValue, out result))
    return result;
return false;
```